### DISCUSSION TITLE
Editable URL slug

### DISCUSSION THREAD

**Discussion by projektorius96** [forem/forem#15866] [Feature Requests]
As far as I know currently it's not possible to edit URL slug (_correct me If I am wrong_) e.g. I've modified one of my post recently whereas my URL have not synched with that any way. I used to blog on Hashnode.com whereby the later one has such option to edit URL slug for a post . Hope so to see one day the same feature on **DEV.to** as well ! Thanks .

**Comment by ildi-x** [comment]
From my experience as a webmaster of a few websites, being able to edit the URL slug is also good SEO practice. Sometimes post titles can be long or contain keywords you dont want in the URL slug. 

**Example Title:** Genius Acquired for $80 Million by MediaLab, Which Is Making Layoffs at Music Lyrics Company 
**Post URL:** https://www.1vibe.com/ildi/genius-acquired-for-80-million-by-medialab-which-is-making-layoffs-at-music-lyrics-company-5251

If I had the option to edit the URL slug for this post, it would be something like this: /genius-acquired-for-80-million-by-medialab

**Comment by Timothy2631** [comment]
I think you are using something different platforms. Because we could easily edit the slug of [zalmi squad schedule](https://cricketcorner.net/peshawar-zalmi-squad-schedule/) articles which is easily. Go to the editing section of the post. And there your will find the slug at the right side of the post. Which will allow you to edit it easily without facing any issue. But after changing the slug you have to reindix it so that the changes could shown.

**Reply by projektorius96** [reply]
Vague solution
